models/review.go

```diff
@@ -612,6 +701,141 @@ func RemoveReviewRequest(issue *Issue, reviewer *User, doer *User) (comment *Com
 	return comment, sess.Commit()
 }
 
+// AddTeamReviewRequest add a review request from one team
+func AddTeamReviewRequest(issue *Issue, reviewer *Team, doer *User) (comment *Comment, err error) {
+	review, err := GetTeamReviewerByIssueIDAndTeamID(issue.ID, reviewer.ID)
+	if err != nil && !IsErrReviewNotExist(err) {
+		return
+	}
+
+	// skip it when reviewer team hase been request to review
+	if review != nil {
+		return nil, nil
+	}
+
+	sess := x.NewSession()
+	defer sess.Close()
+	if err := sess.Begin(); err != nil {
+		return nil, err
+	}
+
+	var official bool
+	official, err = isOfficialReviewerTeam(sess, issue, reviewer)
+
```

Comment:
```suggestion
```

Avoid putting new lines between assigning err and testing err. 

Consider using `if ... ; err != nil { ... }` as this keeps the err test adjacent to its assignment.

---

Further consider whether some log.Error(...) may be appropriate in the handling of some of these errs - plainly returning errs can make tracking down bugs more difficult as you can't work out where the problem originated.

If you're calling multiple named functions it can be helpful to just do a `log.Error("isOfficialReviewerTeam Err: %v", err)` but it's even better if you can give some idea of the arguments that were passed in too.